tests/test_analysis.py

```diff
@@ -155,5 +155,25 @@ def test_analysis(self):
         subprocess.check_call(mpiCmd + cmd)
 
 
+class TestIntersect(unittest.TestCase):
+    def setUp(self):
+        os.chdir(os.path.join(tutorialDir, "intersection"))
```

Comment:
Since we need to make sure that the files are downloaded before any tests are run, we should use the `setUpClass` method instead.